.gitignore

```diff
@@ -165,3 +165,7 @@ cython_debug/
 **/emdat_processed .csv
 .DS_Store
 data/*
+
+# Ignore idata
+*.idata
+**idata**
```

Comment:
This would ignore a file called `process_idata.py`. Instead, ignore all the file extensions that you use for them (`*.idata`, `*.netcdf`, `*.ncdf`, etc)